testing/osde2e/gpu-addon.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+if ! [[ -d  $ARTIFACT_DIR ]] ; then
+    echo "FATAL: osde2e tests requires '/test-run-results' to exist"
+    exit 1
+fi
+
+BURN_RUNTIME_SEC=600
+
+JUNIT_HEADER_TEMPLATE='<?xml version="1.0" encoding="utf-8"?>
+<testsuite errors="NUM_ERRORS" failures="NUM_ERRORS" name="TEST_TARGET" tests="1" time="RUNTIME" timestamp="TIMESTAMP">
+    <testcase name="TEST_TARGET" time="RUNTIME">
+        <CASE_OUTPUT_TAG>
+'
+
+JUNIT_FOOTER_TEMPLATE='
+        </CASE_OUTPUT_TAG>
+    </testcase>
+</testsuite>
+'
+
+function exit_and_abort() {
+    echo "Failed. Aborting."
+    exit 1
+}
+
+function run_test() {
+    TARGET=${1-}
```

Comment:
what about giving the `junit_$NAME` as `$1`? 

is `${1-}` syntax correct, or `${1:-}`?

Reply:
and then:
```
shift
...
RAW_OUTPUT=$(/usr/bin/time -o ${ARTIFACT_DIR}/runtime  ./run_toolbox.py "$@")
```

Reply:
I think that syntax can go ether way.. but changed it to keep same syntax with other scripts.

Regarding the name as $1, I like it less currently.